adlfs/lib.py

```diff
@@ -245,16 +259,25 @@ def call(self, op, path='', **kwargs):
         func = getattr(requests, method)
         url = self.url + path
         try:
+            self.head['x-ms-client-request-id'] = str(uuid.uuid4())
```

Comment:
Minor: Not sure if it matters but the rest of azure is using uuid.uuid1(). If there is a difference can you switch to uuid1()?